detection_rules/etc/version.lock.json

```diff
@@ -359,7 +389,7 @@
         "rule_name": "RC Script Creation",
         "sha256": "8ff8bb29b78a06c2423fd81d4e1ee96b96a55b848136791f25b4415a0ada11f3",
         "type": "eql",
-        "version": 5
+        "version": 7
```

Comment:
Double bump and no SHA256 diff

Reply:
Looks like for some reason the queries have a space diff before the new line character which is causing the diff. As to how this happened is pretty bizarre as the extra spaces are only in 8.3, whereas they do not exist in 8.4, 8.5 or 8.6.

<img width="2331" alt="Screenshot 2023-07-11 at 3 54 13 PM" src="https://github.com/elastic/detection-rules/assets/99630311/e7bea12d-9889-4590-832e-6ee2f070afef">

Reply:
I think the correct way to fix this is to do a one-off adjustment on the 8.3 branch to remove the extra spaces. This should then cause the version lock to identify there are no changes.

Reply:
With changes, ran the version lock and no changes found. So I will push these changes manually to 8.3 branch.

<img width="657" alt="Screenshot 2023-07-11 at 4 23 29 PM" src="https://github.com/elastic/detection-rules/assets/99630311/bb55e868-6805-4f88-a732-f5daed75d2d6">